editor/.eslintrc.json

```diff
@@ -1,3 +1,10 @@
 {
-  "extends": ["next/core-web-vitals", "next/typescript", "prettier"]
+  "extends": ["next/core-web-vitals", "next/typescript", "prettier"],
+  "plugins": ["react-hooks"],
+  "rules": {
+    "react-hooks/rules-of-hooks": "error",
+    "react-hooks/exhaustive-deps": "warn",
+    "no-unused-vars": "off",
+    "@typescript-eslint/no-unused-vars": ["warn"]
```

Comment:
how do folks feel about setting `exhaustive-deps`  and `no-unused-vars` to `error` instead of `warn`? i'd like our rules to be as strict as possible 😇

Reply:
yep, that sounds like a great plan imo. Failed builds would be a great way to avoid React pitfalls and keep the TS game in here as strong as it is right now. 💪